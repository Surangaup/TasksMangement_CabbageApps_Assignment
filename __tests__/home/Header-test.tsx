import * as React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../src/screens/home/header';

test(('Header snapshot'), ()=>{
    const snap = renderer.create(<Header />).toJSON();
    expect(snap).toMatchSnapshot();
})