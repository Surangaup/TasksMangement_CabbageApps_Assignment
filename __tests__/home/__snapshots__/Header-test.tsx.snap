// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "margin": 10,
    }
  }
>
  <View
    style={
      Object {
        "flex": 7,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#ABA2B0",
          "fontFamily": "Gilroy SemiBold",
          "fontSize": 23,
        }
      }
    >
      Hello,
    </Text>
    <Text
      style={
        Object {
          "fontFamily": "Gilroy SemiBold",
          "fontSize": 23,
        }
      }
    >
      Adbur Rahman
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "flex-end",
        "flex": 2,
      }
    }
  >
    <Image
      resizeMode="contain"
      source={
        Object {
          "testUri": "../../../src/asserts/images/user.jpg",
        }
      }
      style={
        Object {
          "borderRadius": 50,
          "height": 50,
          "margin": 5,
          "width": 50,
        }
      }
    />
  </View>
</View>
`;
